### ISSUE TITLE
[a11y]: CopyButton announces "Copied!" multiple times when pressing the copy button

### ISSUE THREAD

**Issue by alfeugds** [carbon-design-system/carbon#13163]
### Package

carbon-components-react

### Browser

Chrome

### Package version

3.3.3

### React version

^16.12.0

### Automated testing tool and ruleset

manual testing with NVDA

### Assistive technology

NVDA

### Description

The `<CopyButton>` component announces "Copied!" message more than once when pressed.

### WCAG 2.1 Violation

1.3.1 

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-copybutton--default

### Steps to reproduce

- Using Google Chrome, open any of the below links where CopyButton is present:
  - https://carbondesignsystem.com/components/code-snippet/usage/#live-demo
  - https://codesandbox.io/s/c681ih
  - https://react.carbondesignsystem.com/?path=/story/components-copybutton--default
- Open NVDA and its speech viewer tool
- Using Tab navigation, navigate until reaching an instance of `<CopyButton>` component and press space bar to act on it.
- Notice that the "Copied!" message is announced more than once.

Related issues:
https://github.com/carbon-design-system/carbon/issues/5385
https://github.com/carbon-design-system/carbon/issues/5553

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems
